### ISSUE TITLE
Does not ask for credentials (Ubuntu 20.04)

### ISSUE THREAD

**Issue by vinsgithub** [ohwgiles/NetworkManager-f5vpn#2]
I had following issues that I solved

1. cmake wants libcurl.so.4 from x86_64-linux-gnu, so changing the link it worked 
```
sudo rm /usr/local/lib/libcurl.so.4
sudo ln -s /usr/lib/x86_64-linux-gnu/libcurl.so.4.6.0 /usr/local/lib/libcurl.so.4
```
2. launching GUI I got "file "libnm-vpn-plugin-f5vpn.so" not found. Did you install the client package?", changing absolute path in "/usr/lib/NetworkManager/VPN/nm-f5vpn-service.name" as 
```
[libnm]
plugin=/usr/lib/NetworkManager/libnm-vpn-plugin-f5vpn.so
```
I got plugin showing in network manager.

Unfortunately there's no way to set credentials and launching connection nothing appears but a error popup "Enter credentials for XXXXXX server"

**Comment by ohwgiles** [comment]
> so changing the link it worked

This should not have been necessary - normally there is nothing in `/usr/local` and CMake should find the library under `/usr/lib`.

> changing absolute path in...

Hmm. I don't know why this should have been needed. NetworkManager is supposed to search standard locations for the plugins. Maybe this is different on Ubuntu. Do other plugins like OpenVPN also include the full path on your system?

> Unfortunately there's no way to set credentials and launching connection nothing appears but a error popup "Enter credentials for XXXXXX server"

In the current implementation, credentials are only requested after trying to connect (because we don't know what type of credentials will be demanded from the site). The implementation just searches the HTML response of an HTTP GET request to the hostname. Perhaps your server's response is just different to the ones I have tested, there seems to be a variety.

You could check the HTML when you go there in a browser and compare it with [this part of the code](https://github.com/ohwgiles/NetworkManager-f5vpn/blob/master/lib/f5vpn_auth.c#L441).

**Comment by ohwgiles** [comment]
Considering this closed by #8
